app/controllers/users_controller.rb

```diff
@@ -130,20 +127,18 @@ def self.do_daily_recipes(all_recipes)
 		return daily_recipes
 	end
 
-	def self.calc_calories(gender, weight, height, age, exercise, goal)
-		#These calculations require kg and cm.  Will need to add
-		#units to the form in the future.
+	def self.calc_calories(preference_list)
```

Comment:
Nice refactoring! This function definitely needed its number of parameters to be reduced